cmd/aries-js-worker/src/aries.js

```diff
@@ -107,22 +125,27 @@ export const Aries = function(opts) {
             return invoke("aries", "Stop", "{}", "timeout while stopping aries")
         },
 
-        waitForNotification : async function waitForNotification(topics) {
-            return new Promise((resolve, reject) => {
-                const timer = setTimeout(_ => resolve(), 10000)
-                // subscribe for all by default if topics not provided
-                if (topics.length == 0){
-                    topics = ["all"]
+        startNotifier : function(callback, topics) {
```

Comment:
Is there a TODO for stop?

Reply:
Yes, will add a TODO and will add stop functionality